.changeset/lazy-foxes-build.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': patch
+---
+
+Added signIn and authHandler resolver for oAuth2 provider
```

Comment:
OIDC* x)

Reply:
Oh thanks :) just copied some features from oauth2 :) I will fix it since oidc is similar to oauth2 :)